packages/FlexGrid/Col/Col.jsx

```diff
@@ -5,42 +5,65 @@ import { Subscriber } from 'react-broadcast'
 import { Col as ReactFlexboxGridCol } from 'react-flexbox-grid'
 
 import safeRest from '../../../shared/utils/safeRest'
+import { deprecate } from '../../../shared/utils/warn'
 import styles from './Col.modules.scss'
 
-const removeProps = ({
-  xs,
-  sm,
-  md,
-  lg,
-  xl,
-  xsOffset,
-  smOffset,
-  mdOffset,
-  lgOffset,
-  xlOffset,
-  ...rest
-}) => safeRest(rest)
-
-const Col = ({ span, offset, children, ...rest }) => (
-  <Subscriber channel="flex-grid">
-    {gutter => (
-      <ReactFlexboxGridCol
-        {...removeProps(rest)}
-        xs={span || true}
-        xsOffset={offset}
-        className={gutter ? styles.padding : styles.gutterless}
-      >
-        {children}
-      </ReactFlexboxGridCol>
-    )}
-  </Subscriber>
-)
+const removeProps = ({ xsOffset, smOffset, mdOffset, lgOffset, xlOffset, ...rest }) =>
+  safeRest(rest)
+
+const Col = ({ span, offset, xs, sm, md, lg, xl, children, ...rest }) => {
+  if (span) {
+    deprecate(
+      'core-flex-grid',
+      'The span prop is deprecated due to the addition of the xs, sm, md, lg and xl responsive props. Please replace the span prop in your Col definition with the xs prop for identical functionality.'
+    )
+  }
+
+  return (
+    <Subscriber channel="flex-grid">
+      {gutter => (
+        <ReactFlexboxGridCol
+          {...removeProps(rest)}
+          xs={xs || span || true}
+          sm={sm || 0}
+          md={md || 0}
+          lg={lg || 0}
+          xl={xl || 0}
+          xsOffset={offset}
```

Comment:
Maybe we should just spread all of the user supplied props directly into the react-flexbox-grid component? We are not doing anything with them, and our API is essentially the same now as the react-flexbox-grid component. The only "extra" thing we have now is the `span` prop, which can be dealt with separately.